partner_dtp/__openerp__.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+
+{
+    "name": 'Partner Days to Pay',
+    "version": '1.7',
+    "author": 'Ursa Information Systems',
```

Comment:
I no longer am part of the savoirfairelinux group, someone else will have to do that.